### PR TITLE
Add jobs to CI build

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
PR that adds CI-build jobs to this repo

**How to test build jobs: **
- [x] open the travis build checks

**Expected outcome: **
- [x] Several build jobs built 
- [x] Screenshot

**How to test prod install: **
- [ ] deploy in prod on clinical-db

**Expected outcome: **
- [ ] no pysam error

**How to test path fix: **
- [ ] import MAF

**Expected outcome: **
- [ ] no path error

- [x] Reviewed @b4ckm4n 
- [ ] Tested @patrikgrenfeldt @b4ckm4n 
- [x] merge & deploy @patrikgrenfeldt 

This is a minor bump since we add CI func but we don't add any end user func

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -16,3 +16,5 @@ pycodestyle
 yamllint
 docutils
 html-linter
+black
+pre-commit
```
Comment (ingkebil) [inline requirements-dev.txt]:
```suggestion
pre-commit
coverage<5
```

**Comment by patrikgrenfeldt** [comment]
Test results: 
image
image

**Comment by patrikgrenfeldt** [comment]
image
This is expected until we merge this PR 👍

**Comment by patrikgrenfeldt** [comment]
Test result pysam:
image
👍

**Pull request merged**